test/specs/commonTests.js

```diff
@@ -468,6 +468,11 @@ export const rendersChildren = (Component, options = {}) => {
     shallow(createElement(Component, requiredProps, child))
       .should.contain(child)
   })
+
```

Comment:
I removed check if Component is `Embed` from tests and replaced it with `createElement` method
Thanks @layershifter for help